src/Generator/Writer.cs

```diff
@@ -50,67 +51,104 @@ public static void Write(OutputData data)
             }
         }
 
-        private static void WriteNativeFunctions(
-            string directoryPath,
-            string glNamespace,
-            Dictionary<string, GLVendorFunctions> groups,
-            Dictionary<NativeFunction, FunctionDocumentation> documentation)
+        private static void WriteFunctionPointers(string directoryPath, List<NativeFunction> nativeFunctions)
         {
-            using StreamWriter stream = File.CreateText(Path.Combine(directoryPath, "GL.Native.cs"));
+            using StreamWriter stream = File.CreateText(Path.Combine(directoryPath, "GL.Pointers.cs"));
             using IndentedTextWriter writer = new IndentedTextWriter(stream);
+
             writer.WriteLine("// This file is auto generated, do not edit.");
             writer.WriteLine("using System;");
             writer.WriteLine("using System.Runtime.InteropServices;");
             writer.WriteLine("using OpenTK.Graphics;");
             writer.WriteLine();
-            writer.WriteLine($"namespace {GraphicsNamespace}.{glNamespace}");
+            writer.WriteLine($"namespace {GraphicsNamespace}");
             using (writer.CsScope())
             {
-                writer.WriteLine($"public static unsafe partial class GL");
+                writer.WriteLine($"public static unsafe partial class GLNative");
                 using (writer.CsScope())
                 {
-                    foreach (var (vendor, group) in groups)
+                    foreach (NativeFunction function in nativeFunctions)
                     {
-                        CsScope? scope = null;
-                        if (!string.IsNullOrWhiteSpace(vendor))
-                        {
-                            writer.WriteLine($"public static unsafe partial class {vendor}");
-                            scope = writer.CsScope();
-                        }
+                        WriteNativeMethod(writer, function, false, null);
```

Comment:
Why are we no longer applying the postfix to the native functions that should have a postfix?